helm/cert-exporter-chart/Chart.yaml

```diff
@@ -1,2 +1,2 @@
 name: cert-exporter-chart
-version: 1.0.0-[[ .SHA ]]
+version: 1.0.0
```

Comment:
The `-[[ .SHA ]]` templating needs to be removed because otherwise we get 422 errors when pushing the chart to Quay.